### DISCUSSION TITLE
Change button text to "Update Connection" (from "Modify Connection")

### DISCUSSION THREAD

**Discussion by ReessKennedy** [nocodb/nocodb#9840] [General]
"Update" or "Save" is the common and standard English word used for updating an existing setting. The current use of "Modify Connection" would be better if switched to "Update Connection" of "Save" because, at least to me, modify in this context looked like a method to initiate the editing process and not to finalize it. As a result of this I actually did not understand many times that "Modify" actually was the method to "Save" or "Update" and lost some time clicking around until I figured it out. 
<img width="386" alt="Screenshot 2024-11-17 at 10 55 53 AM" src="https://github.com/user-attachments/assets/c0d7639b-82e4-4a75-b6c5-c46e881b2b0b">

**Comment by dstala** [comment]
Thanks @ReessKennedy. We will fix it in next release
https://github.com/nocodb/nocodb/issues/9841

**Reply by ReessKennedy** [reply]
Nice!
